src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -125,6 +127,7 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
             IEnumerable<KeyValuePair<string, ReportDiagnostic>> specificDiagnosticOptions,
             bool concurrentBuild,
             bool extendedCustomDebugInformation,
+            bool debugPlusMode,
```

Comment:
This method has 20+ parameters. None documented here.

Most of these parameters can be traced to being passed to compilation or to emit and that is where they are generally documented. This one is documented in couple places in codegen, but it is just hard to document it at every entry point.